# ros2 param load /thermal_camera_to_world /home/yuan/server_ws/src/thermal_camera2world/config/camera2world.yaml


/thermal_camera_to_world:
  ros__parameters:
    World_UpperLeft: [4.8, 2.9]
    World_UpperRight: [5.3, 7.5]
    World_LowerRight: [7.8, 7.2]
    World_LowerLeft: [6.5, 0.2]
    Threshold_Temperature: 60.0
    Alert_Waiting_Time: 5

